Makefile

```diff
@@ -89,7 +89,7 @@ run: $(ISO)
 debug: ## build and run the OS in debug mode
 debug: CFLAGS += $(DEBUG_CFLAGS)
 debug: $(ISO)
-	$(QEMU) -cdrom $< -serial file:/tmp/serial.log
+	$(QEMU) -cdrom $< -serial file:/tmp/serial.log -m 1G
```

Comment:
Userland gets invalid opcode again if I don't increase above 256MB of RAM... I wonder why this is happening. Maybe we're exceeding the RAM usage? I did these changes in the ELF loader fixes branch first which is why I increased further.

Reply:
It could be a problem with memory mapping but I am not sure. I was working on recursive mapping but it did not quite work..